devtools/conda-envs/no_openeye.yaml

```diff
@@ -63,8 +63,5 @@ dependencies:
   - codecov
   - requests-mock
 
-  # Temporary branches
-  - pip:
-    - git+https://github.com/openforcefield/openff-toolkit.git@main
-    - git+https://github.com/yoshanuikabundi/openff-qcsubmit.git@toolkit_011_support
-    - git+https://github.com/Yoshanuikabundi/QCEngine.git@off_toolkit_011_support
+  - importlib-metadata >=4
+  - importlib_metadata >=4
```

Comment:
```suggestion
  - importlib-metadata >=4
```

Duplicate

Reply:
Sure, not a typo but no longer strictly necessary to include here

https://github.com/openforcefield/openff-bespokefit/pull/202#issuecomment-1314112832
https://github.com/conda-forge/openff-toolkit-feedstock/pull/37